examples/java/src/main/java/org/apache/beam/examples/DebuggingWordCount.java

```diff
@@ -62,12 +62,14 @@
  * }
  * </pre>
  *
- * <p>To execute this pipeline using the Dataflow service and the additional logging discussed
- * below, specify pipeline configuration:
+ * <p>To change the runner, specify:
+ * <pre>{@code
+ *   --runner=YOUR_SELECTED_RUNNER
+ * }
+ * </pre>
+ *
+ * <p>To use the additional logging discussed below, specify:
```

Comment:
I think DebuggingWordCount is designed to be Dataflow specific.
We need to re-write it or remove it.